jit/analyze.ts

```diff
@@ -90,7 +90,7 @@ module J2ME {
       return true;
     }
     // XXX Determine whether we can start using the code in this function.
-    return false;
+    return methodInfo.isFinal;
     var result = methodInfo.isFinal;
```

Comment:
Not yet, that code is only valid if every class in every jar is loaded. However, mbx and I have been chatting on enabling better analysis.